lib/tuya.js

```diff
@@ -1460,6 +1460,22 @@ const valueConverter = {
     },
     inverse: {to: (v) => !v, from: (v) => !v},
     onOffNotStrict: {from: (v) => v ? 'ON' : 'OFF', to: (v) => v === 'ON'},
+    errorBatteryLow: {
+        from: (v) => {
+            if (v === 0) return {'battery_low': false};
+            if (v === 1) return {'battery_low': true};
+            return {'error': v};
```

Comment:
Can we make error a boolean? (or is it an error code?)

Reply:
It sends bitmap datatype, so I'm not sure we can make it a boolean. It seems like it's an error code. I didn't see any other data than 0 or 1.